download_cmd.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/facebookgo/stackerr"
+	"github.com/spf13/cobra"
+)
+
+type downloadCmd struct{}
+
+type releaseFiles struct {
+	UserFiles map[string]map[string]string `json:"userFiles"`
+	Checksums map[string]map[string]string `json:"checksums"`
+}
+
+type fileMap map[string][]byte
+
+func (d *downloadCmd) writeFiles(files fileMap) error {
+	for path, data := range files {
+		dir := filepath.Dir(path)
+		err := os.MkdirAll(dir, os.ModeDir|os.ModePerm)
+		if err != nil {
+			return stackerr.Wrap(err)
+		}
+		file, err := os.Create(path)
+		defer file.Close()
+		if err != nil {
+			return stackerr.Wrap(err)
+		}
+		_, err = file.Write(data)
+		if err != nil {
+			return stackerr.Wrap(err)
+		}
+	}
+	return nil
+}
+
+func (d *downloadCmd) downloadScripts(e *env, release releaseFiles, files fileMap) error {
```

Comment:
also look at deploy to see how you can parallelize this operation. and follow the same pattern here as well.
not sure why you are downloading things in a serial manner.